src/Compiler/Checking/AugmentTypeDefinitions.fs

```diff
@@ -0,0 +1,1617 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+///  Generate the hash/compare functions we add to user-defined types by default.
```

Comment:
I understand the good intention behind renaming this file, but is there a tip to make this more diff-review-friendly?

Reply:
Thanks kerams. I will gather my brain capacity and review this early next week.